pysatNASA/instruments/ses14_gold.py

```diff
@@ -115,7 +115,7 @@ def clean(self):
 
 # Set the list_files routine
 fname = ''.join(('gold_l2_{tag:s}_{{year:04d}}_{{day:03d}}_v{{version:02d}}',
-                 '_r{{revision:02d}}_c{{cycle:02d}}.nc'))
+                 '_r{{revision:02d}}_c??.nc'))
```

Comment:
This should not have been changed.  We want to continue to support cycle here.

Reply:
Yay.....

Reply:
just pushed a branch now